xbmc/windowing/X11/WinSystemX11GLContext.cpp

```diff
@@ -233,3 +236,18 @@ std::unique_ptr<CVideoSync> CWinSystemX11GLContext::GetVideoSync(void *clock)
   }
   return pVSync;
 }
+
+void* CWinSystemX11GLContext::GetVaDisplay()
+{
+#if defined(HAVE_LIBVA)
+  if (dynamic_cast<CGLContextEGL*>(m_pGLContext))
```

Comment:
Hm, where is CheckCaps called? I couldn't find a call to CheckCaps() anywhere and when I put a std::terminate() in there nothing happened.